index.js

```diff
@@ -46,20 +150,27 @@ SASSPlugin.prototype.toTree = function(tree, inputPath, outputPath, inputOptions
     }
   }
 
-  var SassCompiler = SassCompilerFactory(options.implementation);
   var ext = options.extension || 'scss';
   var paths = options.outputPaths;
-  var trees = Object.keys(paths).map(function(file) {
+
+  var inputOutputMap = Object.keys(paths).map(function(file) {
     var input = path.join(inputPath, file + '.' + ext);
     var output = paths[file];
-    return new SassCompiler(inputTrees, input, output, options);
+
+    return { input, output };
   });
 
+  var trees = [
+    new SassCompiler(new Funnel(mergeTrees(inputTrees), {
+      include: ['**/*.scss', '**/*.sass', '**/*.css'],
+    }), inputOutputMap, options)
+  ];
+
   if (options.passthrough) {
-    trees.push(new Funnel(tree, options.passthrough));
+    inputTrees.push(new Funnel(tree, options.passthrough));
   }
 
-  return mergeTrees(trees);
+  return mergeTrees(trees, { overwrite: true });
```

Comment:
FWIW, we should probably only do a `mergeTrees` if we have more than one `inputTrees`.